main/utils/environment.ts

```diff
@@ -0,0 +1,13 @@
+import electron from 'electron';
+
+export const isDevelopment = () => {
+  if (typeof electron === 'string') {
+    throw new TypeError('Not running in an Electron environment!');
+  }
+
+  const isEnvSet = 'ELECTRON_IS_DEV' in process.env;
+  const getFromEnv =
+    Number.parseInt(process.env.ELECTRON_IS_DEV!, 10) === 1;
```

Comment:
I'm not sure we use `ELECTRON_IS_DEV`, it might save a lot of time and space to just use the `electron.app.isPackaged` method everywhere we need this. Less complexity too.